beacon-chain/sync/initial-sync/blocks_fetcher_utils.go

```diff
@@ -420,54 +391,6 @@ func uint64MapToSortedSlice(input map[uint64]bool) []uint64 {
 	return output
 }
 
-// `filterPeerWhichCustodyAtLeastOneDataColumn` filters peers which custody at least one data column
-// specified in `neededDataColumns`. It returns also a list of descriptions for non admissible peers.
-func filterPeerWhichCustodyAtLeastOneDataColumn(
```

Comment:
These are linter spotted issues, it's not used anymore, however not sure why it's not caught by CI in https://github.com/prysmaticlabs/prysm/pull/14977/files#diff-c81d6a0f926408e8bfafff8493bd30f16f486afd6d38b9582a8255c90c464f04